api/src/pcapi/core/offers/models.py

```diff
@@ -238,7 +238,7 @@ def remainingQuantity(cls) -> Case:  # pylint: disable=no-self-argument
         return sa.case([(cls.quantity.is_(None), None)], else_=(cls.quantity - cls.dnBookedQuantity))
 
     AUTOMATICALLY_USED_SUBCATEGORIES = [
-        subcategories_v2.ABO_MUSEE.id,
+        subcategories_v2.ABO_MUSEE.id,  # FIXME: ogeber 2023-10-26: delete line cf. ticket
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Je veux bien le numéro du ticket dans le commentaire :pray: